examples/sampling/pyg/test.py

```diff
@@ -0,0 +1,191 @@
+import dgl.graphbolt as gb
+import torch
+import torch.nn.functional as F
+from torch_geometric.data import Data
+from torch_geometric.loader import NeighborSampler
+from torch_geometric.nn import SAGEConv
+
+
+class GraphSAGE(torch.nn.Module):
+    def __init__(self, in_channels, hidden_channels, out_channels):
+        super(GraphSAGE, self).__init__()
+        self.conv1 = SAGEConv(in_channels, hidden_channels)
+        self.conv2 = SAGEConv(hidden_channels, out_channels)
+
+    def forward(self, x, adjs):
+        for i, (edge_index, _, size) in enumerate(adjs):
```

Comment:
`adj_index` or some other variable name instead of `i`

Reply:
got it

Reply:
working on it